ci/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
```

Comment:
I considered that, but decided against it because the Intel oneAPI image is massive and contains the full toolkit which we don't need. Just the compiler is sufficient for this project. I wouldn't mind using the massive image if we could do that out of the box, but we do need to build our own image to provide SDL2, so I opted for a slim but still comprehensive starting point.